src/build_info.cpp

```diff
@@ -298,17 +294,9 @@ version_table_manager::version_table_manager()
 #endif
 
 #ifdef __APPLE__
-
+    // Always compiled in.
```

Comment:
Simply remove this from the feature list, then. It's intended for optional features, not mandatory ones.

Reply:
Windows has it same way, isn't it?

Reply:
Ah, you're right. OK, it's better to keep Cocoa notifications in the list.